tests/sandbox/linalg/test_linalg.py

```diff
@@ -152,3 +154,96 @@ def test_matrix_inverse_solve():
     node = matrix_inverse(A).dot(b).owner
     [out] = inv_as_solve.transform(None, node)
     assert isinstance(out.owner.op, Solve)
+
+
+def test_cholesky_dot_lower():
+    cholesky_lower = Cholesky(lower=True)
+    cholesky_upper = Cholesky(lower=False)
+
+    L = matrix("L")
+    L.tag.lower_triangular = True
+
+    C = cholesky_lower(L.dot(L.T))
+    f = pytensor.function([L], C)
+
+    if config.mode != "FAST_COMPILE":
```

Comment:
You can make the rewrite always be included with `get_default_mode().including("rewrite_name"), so the test passes in both modes.